docs/storage.md

```diff
@@ -75,7 +75,7 @@ To tune the rate of ingested samples per second, you can either reduce the numbe
 
 If your local storage becomes corrupted for whatever reason, your best bet is to shut down Prometheus and remove the entire storage directory. You can try removing individual block directories, or WAL directory, to resolve the problem, this means losing a time window of around two hours worth of data per block directory. Again, Prometheus's local storage is not meant as durable long-term storage.
 
-CAUTION: Non-POSIX compliant filesystems are not supported by Prometheus' local storage as unrecoverable corruptions may happen. NFS is could be POSIX-compliant, but most implementations are not. Cases of corruption are also reported with other networked filesystems. It is strongly recommended to use a local filesystem for reliability.
+CAUTION: Non-POSIX compliant filesystems are not supported by Prometheus' local storage as unrecoverable corruptions may happen. NFS could be POSIX-compliant, but most implementations are not. Cases of corruption are also reported with other networked filesystems. It is strongly recommended to use a local filesystem for reliability.
```

Comment:
Should we explicitly also call out AWS EFS, as many users of it don't seem to realise it's NFS?

Reply:
Yes, noted.